integrationTests/cypress/integration/cas_rest.feature

```diff
@@ -0,0 +1,47 @@
+Feature: API-based CAS login and logout functionality
+
+  @requires_testuser
+  Scenario: user can access the API with username and password
+    Given the user is logged out of the CES
+    Then the user can access the SonarQube API with username and password
+
+  @requires_testuser
+  Scenario: user can not access the API with wrong username and password
+    Given the user is logged out of the CES
+    Then the user can not access the SonarQube API with wrong username and password
+
+  @requires_testuser
+  @requires_api_token_to_be_removed_afterwards
+  Scenario: API authentication with User Token
+    Given the user is member of the admin user group
+    When the user can create a User Token via the Web API
+    Then the user can access the Web API with the User Token
+
+  @requires_testuser
+  Scenario: /users API responds with correct user attributes
+    When the user's attributes are requested via Web API
+    Then the user's attributes are matching those in the user backend
+
+  Scenario: /users API's respond contains admin group for admin user
+    When the admin user's attributes are requested via Web API
+    Then the user's attributes should include the admin group
```

Comment:
No, because it should not only contain the field, but the field has to match the configured admin group. I think renaming the step would bring more clarity here.